### DISCUSSION TITLE
Could not parse content for http://example.com/scripts/jsstore.worker.min.js.map

### DISCUSSION THREAD

**Discussion by kAvItHa-cL** [ujjwalguptaofficial/JsStore#208] [Q&A]
DevTools failed to load SourceMap: Could not parse content for http://example.com/scripts/jsstore.worker.min.js.map :Unexpected token < in JSON at position 0

i always get this error initially when we load website , once refresh error will not be there but warning remains constantly.

**Comment by ujjwalguptaofficial** [comment]
can't say whats happening.Please provide a POC or link to your site.
